src/test/java/command/MarkCommandTest.java

```diff
@@ -0,0 +1,39 @@
+package command;
```

Comment:
I like it that you even form different packages for test cases! That is a good practice!